ui/src/main/java/org/pentaho/di/ui/trans/steps/tableoutput/TableOutputDialog.java

```diff
@@ -22,11 +22,7 @@
 
 package org.pentaho.di.ui.trans.steps.tableoutput;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Please change this back so that imports are included indivudually, not with .*

Reply:
Changed it back to include the imports. Can you please check now.